doc/code_style/mom.cpp

```diff
@@ -59,34 +59,32 @@
  * Will be parsed by Doxygen.
  */
 struct another_stupid_type {
-	rofl_t blergh_gnampf;  //!< variables and functions in lowercase
-	                       //   with _ as word separator.
-	epictype42_t løl;      //!< no question, we use UTF-8 encoding.
-	ub0rtype røfëlkøptør;  //!< proper documentation is Doxygen parsable.
+	rofl_t blergh_gnampf;     //!< variables and functions in lowercase
+	                          //   with _ as word separator.
+	epictype42_t løl;         //!< no question, we use UTF-8 encoding.
+	ub0rtype     røfëlkøptør; //!< proper documentation is Doxygen parsable.
 };
 
 
-
 /**
  * Yet another Valve game. Go on, nothing to see here.
  */
 class HalfLife3 : public ValveGame {
-// vvv The access modifier is at the same indent level as the class definition:
+	// vvv The access modifier is at the same indent level as the class definition:
 public:
-	void release_game(const ValveTeam &team,
+	void release_game(const ValveTeam& team,
 	                  uint128_t        passed_time,
-	                  const Engine    *game_engine,
-	                  Internet        *community,
-	                  bool             keep_secret=true) {
+	                  const Engine*    game_engine,
+	                  Internet*        community,
+	                  bool             keep_secret = true) {
```

Comment:
like in python for default argument values, there's no spaces around `=`.

this should not be in columns for change-tracking reasons. once the alighment doesn't match, we have to realign all lines even though they did not change. so the spacing should be done with just one space.

Reply:
`SpaceBeforeAssignmentOperators: 'false'`

Need to see if there is a setting for special circumstances for this.